integration-tests/smoke/ccip/ccip_aptos_msghasher_test.go

```diff
@@ -78,6 +111,53 @@ func Test_CCIP_AptosMessageHasher_OnChainVerification(t *testing.T) {
 	})
 }
 
+// Test_CCIP_AptosMessageHasher_FuzzGeneratedCases runs integration tests on fuzz-generated cases
+// The test cases are generated by the FuzzAptosMessageHasher_CreateTestCases function
+// and saved to the testdata/fuzz/FuzzAptosMessageHasher directory.
```

Comment:
The fuzz corpus cases should already be automatically executed as unit tests. We should not have to manually run them.

Reply:
We don't use the fuzzing tool as @makramkd and I discussed that fuzzing does not work well with tests spinning up services in docker.